src/solve.jl

```diff
@@ -162,6 +162,14 @@ function solve!(system::InitialValueProblem{<:HybridSystem,
             # TODO temporary hack, to be resolved in #447
             options_copy[:mode] = "reach"
         end
+
+        if opC isa BFFPS19
```

Comment:
It is better to use dispatch on `solve!`; this clause can be determined by the type of `opC`.

Reply:
I've opened https://github.com/JuliaReach/Reachability.jl/issues/646, which can be handled together with BFFPS19 in a follow-up PR.

Reply:
I fear that this adds code duplication.
We could apply a [visitor pattern](https://en.wikipedia.org/wiki/Visitor_pattern) (call a helper method from here whose default implementation is just empty), but that also feels partially artificial.